widget/src/helpers.rs

```diff
@@ -249,6 +249,38 @@ where
     container(content).center(Length::Fill)
 }
 
+/// Creates a new [`Pin`] widget with the given content.
+///
+/// A [`Pin`] widget positions its contents at some fixed coordinates inside of its boundaries.
```

Comment:
What you mean "inside of its boundaries"? Container will be moves and its boundaries as well, so coordinates also changed. Do you mean boundaries of the parent container?

Reply:
No, that isn't reasonable. Everything in `iced` works in relative coordinates. This is no exception.

Reply:
You position some child content within the bounds of the `pin` widget. In the provided example, a `Text` widget (really an Element) is placed at coordinates `(50, 50)` within the bounds of the container-like `pin` widget which has width and height of `500`.

Reply:
I think the misunderstanding was because word "its" can refer to different things depending on the interpretation. I read it as
```
A [`Pin`] widget positions its contents ... inside of its boundaries.
                                  ^--------------------'
```
while the author assumed
```
A [`Pin`] widget positions its contents ... inside of its boundaries.
             ^-----------------------------------------'
```
I'm not sure if this kind of misinterpretation is possible for native speakers, maybe from their point of view this is pretty straightforward sentence. I just like to highlight that people with other background may read it incorrectly.

Reply:
I'm not a native speaker either, but I think your original interpretation would only be appropriate for the sentence "A `Pin` widget positions its contents ... inside of _their_ boundaries."

Either way the example (and now this discussion) should clarify any future confusion.

Reply:
Since only one of the interpretations makes absolutely no sense whatsoever... I believe reasonable people will fall back to the sensical one.